applets/wncklet/org.mate.panel.applet.window-list.gschema.xml.in

```diff
@@ -10,6 +10,16 @@
       <summary>Show windows from all workspaces</summary>
       <description>If true, the window list will show windows from all workspaces. Otherwise it will only display windows from the current workspace.</description>
     </key>
+    <key name="show-window-thumbnails" type="b">
+      <default>true</default>
+      <summary>Display window thumbnails on hover</summary>
+      <description>If true, then when hovering over a taskbar item, a thumbnail of the window will appear. It will go away as soon as the mouse leaves the item.</description>
+    </key>
+    <key name="thumbnail-window-size" type="i">
+      <default>200</default>
+      <summary>Size of window thumbnails</summary>
+      <description>Size in pixels of the window preview thumbnail. The largest between width and height will use this value, the other one will be calculated to maintain the correct aspect ratio.</description>
```

Comment:
same here

Reply:
Again, why? I never saw a version in dconf key description.

Reply:
Is there a way to make these conditional during compile time? Maybe on a separate gschema file?